src/utils/dateRange.ts

```diff
@@ -51,3 +51,20 @@ export function getSinceDateRangeString(key: string = 'since_date') {
     startDate,
   });
 }
+
+export function getMonthDate(offset: number) {
+  const today = new Date();
+  today.setMonth(today.getMonth() - offset);
+  const startDate = new Date(today.getFullYear(), today.getMonth(), 1);
+  const endDate = offset === 0 ? new Date() : new Date(today.getFullYear(), today.getMonth() + 1, 0);
+
+  return { start_date: format(startDate, 'yyyy-MM-dd'), end_date: format(endDate, 'yyyy-MM-dd') };
```

Comment:
This can be simplified as:

```
  const start_date = format(startOfMonth(today), 'yyyy-MM-dd');
  const end_date = format(offset ? endOfMonth(today) : today, 'yyyy-MM-dd');
```

Reply:
I can change the variable names, but i kept them so they would follow the "Camel Case"
 naming concepts. When be used in the props as args they are then used with "snake case"